--- src/main/java/com/savagevegetables/outbreak/SavageVegetablesOutbreak.java
+++ src/main/java/com/savagevegetables/outbreak/SavageVegetablesOutbreak.java
@@ -41,6 +41,12 @@
         event.put(EntityInit.CELERY_ENTITY.get(), com.savagevegetables.outbreak.entity.mob.boss.CeleryEntity.createBossAttributes().build());
         event.put(EntityInit.EGGPLANT_ENTITY.get(), com.savagevegetables.outbreak.entity.mob.stage2.EggplantEntity.createAttributes().build());
         event.put(EntityInit.ZUCCHINI_ENTITY.get(), com.savagevegetables.outbreak.entity.mob.stage2.ZucchiniEntity.createAttributes().build());
+        event.put(EntityInit.LEEK_ENTITY.get(), com.savagevegetables.outbreak.entity.mob.vegetables.LeekEntity.createAttributes().build());
+        event.put(EntityInit.CUCUMBER_ENTITY.get(), com.savagevegetables.outbreak.entity.mob.vegetables.CucumberEntity.createAttributes().build());
+        event.put(EntityInit.POTATO_ENTITY.get(), com.savagevegetables.outbreak.entity.mob.vegetables.PotatoEntity.createAttributes().build());
+        event.put(EntityInit.CARROT_ENTITY.get(), com.savagevegetables.outbreak.entity.mob.vegetables.CarrotEntity.createAttributes().build());
+        event.put(EntityInit.BROCCOLI_ENTITY.get(), com.savagevegetables.outbreak.entity.mob.vegetables.BroccoliEntity.createAttributes().build());
+        event.put(EntityInit.RADISH_ENTITY.get(), com.savagevegetables.outbreak.entity.mob.vegetables.RadishEntity.createAttributes().build());
     }
 }
